tnt/src/main/java/org/dcsa/conformance/standards/tnt/action/TntGetEventsAction.java

```diff
@@ -62,6 +69,12 @@ protected Stream<? extends ConformanceCheck> createSubChecks() {
   }
 
   public ObjectNode asJsonNode() {
-    return super.asJsonNode().set("suppliedScenarioParameters", sspSupplier.get().toJson());
+    var dsp = getDspSupplier().get();
+    ObjectNode jsonActionNode = super.asJsonNode().set("suppliedScenarioParameters", sspSupplier.get().toJson());
+    String cursor = dsp.cursor();
+    if (cursor != null && !cursor.isEmpty()) {
```

Comment:
You can make it easier for yourself (and the reviewers), by just using methods from `StringUtils`, like `StringUtils.isNotEmpty` or `StringUtils.isNotBlank`, depending on your needs (check Javadoc).